doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst

```diff
@@ -1667,17 +1670,17 @@ VX Parameters for Observations
    The script ``ush/get_obs.py`` contains further details on the files and
    directory structure of each obs type.
 
-``[CCPA|NOHRSC|MRMS|NDAS|AERONET|AIRNOW]_OBS_AVAIL_INTVL_HRS``: (Defaults: [1|6|1|1|24|1])
+``[CCPA|NOHRSC|MRMS|NDAS|AERONET|AIRNOW|GOESAOD|GOESADP]_OBS_AVAIL_INTVL_HRS``: (Defaults: [1|6|1|1|24|1])
   Time interval (in hours) at which the various types of obs are available
   on NOAA's HPSS. 
 
-  Note that MRMS files are in fact available every few minutes, but here
+  Note that MRMS and GOES files are in fact available every few minutes, but here
   we set the obs availability interval to 1 hour because currently that
   is the shortest output interval for forecasts, i.e. the forecasts cannot
   (yet) support sub-hourly output.
 
-``[CCPA|NOHRSC|MRMS|NDAS|AERONET|AIRNOW]_OBS_DIR``: (Default: ``"{{ workflow.EXPTDIR }}/obs_data/[ccpa|nohrsc|mrms|ndas|aeronet|airnow]"``)
-   Base directory in which CCPA, NOHRSC, MRMS, NDAS, AERONET, or AIRNOW obs files needed by
+``[CCPA|NOHRSC|MRMS|NDAS|AERONET|AIRNOW|GOESAOD|GOESADP]_OBS_DIR``: (Default: ``"{{ workflow.EXPTDIR }}/obs_data/[ccpa|nohrsc|mrms|ndas|aeronet|airnow]"``)
```

Comment:
@mkavulich Update the `Defaults: ...` here as well.

Reply:
Added a description above

Reply:
Thanks!